libscript/src/script-module.cpp

```diff
@@ -1265,8 +1264,24 @@ bool MCScriptWriteInterfaceOfModule(MCScriptModuleRef self, MCStreamRef stream)
                     }
                     break;
                     case kMCScriptTypeKindRecord:
-                        // TODO - Records not yet supported
-                        break;
+                    {
+                        auto t_record_type =
```

Comment:
Do we have an Official Pronouncement yet on whether it's okay to use `auto` where the type is initialisation from `static_cast<>` or similar situations where without `auto` you'd need to write out the same type in a single line of code?

There doesn't seem to be any urgency to merge this so I'm happy to wait until there's a consensus.